pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_comparison-between-modules-and-DITA-topics.adoc

```diff
@@ -1,14 +1,13 @@
-include::../attributes.adoc[]
 [id='comparison-between-modules-and-DITA-topics_{context}']
 = Comparison between modules and DITA topics
 
-A module is similar to a topic in Darwin Information Typing Architecture (DITA) topic. 
+A module is similar to a topic in Darwin Information Typing Architecture (DITA). 
 
 A module differs from a DITA-topic in the following aspects:
 
 * Source language
 + 
-Modules are written AsciiDoc, whereas DITA topics are written in xml.
+Modules are written AsciiDoc, whereas DITA topics are written in XML.
 
 * Structure
 +
```

Comment:
For some reason, I can't add comments below this line... @benradey, any idea why that might be?

Anyway, my comments:

"Following is the comparison..." You can just change this to a title for the table. Alternatively, "the following table compares the structure of a DITA topic and a module."

I think that this table is kind of misleading, though...I'm not real sure what you're trying to show here. A module, ideally, should be structured like a DITA topic, even though it doesn't use XML. There's no meaningful difference between `<title>Topic title</title>` and `= Topic title`. And you should ideally have a short description, body, and related links.



Also, technically, you can have nested topics within a topic. You're just not supposed to according to the Mod Docs guidelines.